### PR TITLE
[FUSEQE-3846] Backport to master

### PR DESCRIPTION
**Pull request by AnetaCadova**
//test: `@docs-verification`
`@ENTESB-13763`

### CODE REVIEW

**Comment by avano** [comment]
retest this please

**Comment by AnetaCadova** [comment]
This test will fail until https://issues.redhat.com/browse/ENTESB-13763 is resolved.

**Comment by syndesisqe** [comment]
Following scenarios failed:
```
Connection - CRUD | Create and delete 
Connection - CRUD | Kebab menu 
Connection - CRUD | Kebab menu edit and view 
Documentation verification | Version check 
```

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

File: ui-tests/src/test/java/io/syndesis/qe/steps/other/DocsVerificationSteps.java [hunk 1]
```diff
@@ -12,31 +12,26 @@
...
 
 public class DocsVerificationSteps {
-    private HashMap<String, String> versions = new HashMap<>();
-    private static String versionOnAboutPage;
-
-    public DocsVerificationSteps() {
-        versions.put("1.9", "7.6");
-    }
+    final String syndesisVersion = "1.10";
+    final String docsVersion = "7.7";
```
Comment (avano) [inline ui-tests/src/test/java/io/syndesis/qe/steps/other/DocsVerificationSteps.java:19]: in master you probably want to have `2.0` and `8.0` :)

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Pull request merged**